docs/source/backends/zerotier.rst

```diff
@@ -274,30 +274,28 @@ Client specific settings
-| ``config_path``        | string  |              | path to the persistent configuration folder                                                        |
-+------------------------+---------+--------------+----------------------------------------------------------------------------------------------------+
-| ``copy_config_path``   | string  | ``'0'``      | specifies whether to copy the configuration file to RAM                                            |
-|                        |         |              |                                                                                                    |
-|                        |         |              | ``'0'`` - No, ``'1'`` - Yes, this prevents writing to flash in zerotier controller mode            |
-+------------------------+---------+--------------+----------------------------------------------------------------------------------------------------+
-| ``port``               | integer | ``9993``     | port number of the zerotier service                                                                |
-+------------------------+---------+--------------+----------------------------------------------------------------------------------------------------+
-| ``local_conf``         | boolean |              | path of the local zerotier configuration                                                           |
-+------------------------+---------+--------------+----------------------------------------------------------------------------------------------------+
-| ``secret``             | boolean |              | secret key of the zerotier client (network member), leave it blank to be automatically determined  |
-+------------------------+---------+--------------+----------------------------------------------------------------------------------------------------+
+* nwid_ifname
+
++------------------------+---------+----------------------------+------------------------------------------------------------------------------------------------------+
+| key name               | type    | default                    | description                                                                                          |
++========================+=========+============================+======================================================================================================+
+| ``name``               | string  |  ``ow_zt``                 | name of the zerotier network                                                                         |
++------------------------+---------+----------------------------+------------------------------------------------------------------------------------------------------+
+| ``nwid_ifname``        | list    | ``[{}]``                   | list of dictionaries containing strings with **16-digit** hexadecimal network IDs for joining,       |
```

Comment:
I am not a fan of this name, can we find a more descriptive name?
Why don't we simply call this `networks`?

Reply:
Sure, that looks clean and is more descriptive.